drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -940,61 +939,81 @@ def report_wcs(total_product_list, json_timestamp=None, json_time_since_epoch=No
                 # Restore an alternate to be the primary WCS
                 for key, value in dict_of_wcskeys_names.items():
                     if key != ' ':
-                        wcsutil.altwcs.restoreWCS(edp_object.full_filename, ext=extname_list, wcskey=key)
                         alt_key = key
-                        alt_wcs_name = value
+                        alt_wcs_name = value.strip()
+
+                        if len(alt_wcs_name) == 13 and alt_wcs_name.startswith('IDC_'):
```

Comment:
We currently have 9 character rootnames for the IDCTAB's leading to 13 character long IDC_* WCSNAME.  However, that relies on an assumption which could be broken even if only by the end user.  I believe a safer way to detect the default WCS would be to look for:

    '-' not in alt_wcs_name and alt_wcs_name.startswith('IDC_')

That makes no assumptions about filename length while eliminating OPUS and apriori/aposteriori solutions which add the '-*' terms at the end.

Reply:
Much better solution.  Fix seen on next commit.